### ISSUE TITLE
Requires yarn to build

### ISSUE THREAD

**Issue by Zhincore** [magicdawn/node-mozjpeg#1]
When installing this library i get `yarn: not found` error. I don't use yarn, I use npm. Is it possible to make this project work independently of package manager?

Full error:
```
npm ERR! path /[redacted]/node_modules/node-mozjpeg
npm ERR! command failed
npm ERR! command sh -c prebuild-install --runtime=napi || yarn build-native
npm ERR! sh: 1: yarn: not found
```

**Comment by magicdawn** [comment]
"install": "prebuild-install --runtime=napi || yarn build-native",
yarn is used to run scripts.

btw, there are prebuilds already exists in 
https://cdn.jsdelivr.net/gh/magicdawn/prebuild-binary@master/files/node-mozjpeg/v0.4.1/

normally you will not need that

**Comment by magicdawn** [comment]
fixed in v1.0.1
